merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/Duration.java

```diff
@@ -496,16 +527,15 @@ public boolean isZero() {
   }
 
   public boolean isEqualTo(final Duration other) {
+    if(other == null) return false;
     return this.durationInMicroseconds == other.durationInMicroseconds;
   }
 
   /** Determine whether two durations are the same. */
   @Override
   @Deprecated
   public boolean equals(final Object o) {
-    if (!(o instanceof Duration)) return false;
-    final var other = (Duration)o;
-
+    if (!(o instanceof final Duration other)) return false;
```

Comment:
`Duration.isEqualTo`, which this method calls. This is an override of a default Object method, though, so we can't remove it or the `deprecated` tag.

Reply:
FYI in procedural scheduling branch I have converted `Duration` to a record. So I deleted this function, because it is automatically implemented (and not deprecated).